README.md

```diff
@@ -28,12 +28,48 @@ When the reader has completed this Code Pattern, they will understand how to:
 * [Flask](http://flask.pocoo.org/): is a micro web framework.
 * [ngrok](http://ngrok.com/): allows you to open an HTTPS tunnel into your REST API.
 
-## Prerequisites:
+## Featured technologies
+
+* [Artificial Intelligence](https://medium.com/ibm-data-science-experience): Artificial intelligence can be applied to disparate solution spaces to deliver disruptive technologies.
+* [Python](https://www.python.org/): Python is a programming language that lets you work more quickly and integrate your systems more effectively.
+
+## Prerequisites
 
 The following are prerequisites to start developing this application:
 
 * Xcode
 * Turi Create (5.0 Beta 2)
 * Cocoapods
 * Python 2.7 & PIP
+* Flask
 * Jupyter
+* ngrok
+
+# Steps
+
+(NOTE: I still need to add screenshots/pictures to all the steps!)
+
+1. [Download and organize the latest MovieLens dataset](#1-download-and-organize-the-latest-movielens-dataset)
+2. [Train the Turi Create Recommender model](#2-train-the-turi-create-recommender-model)
+3. [Run the backend](#3-run-the-backend)
+4. [Deploy the iOS app](#4-deploy-the-ios-app)
+
+### 1. Download and organize the latest MovieLens dataset
+
+There's a script available in the root directory of this repository, called `setup.sh`. Run this script - it'll download the movielens data, and organize it, for you.
+
+### 2. Train the Turi Create Recommender model
+
+In order for the recommendations to work, you need a trained model. Turi Create is a Python package that will train this model for you.
+
+Go ahead and start your Jupyter Notebooks. In the `local_model_training` folder, open the `TrainRecommendationModel.ipynb` notebook. Run the notebook, and once it's done executing, you're going to have a folder called `movie_rec`, which contains the trained Turi Create model. Unlike other machine learning libraries, Turi Create does not use files for models - it uses folders.
```

Comment:
Same comment with the hyphen.

    does not use files for models; instead, it uses folders.